docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,407 @@
+    - MEDIA_STREAMER_PARAM_WEBRTC_ADD_ICE_CANDIDATE
+    - MEDIA_STREAMER_PARAM_WEBRTC_RTP_TRANSCEIVER_DIRECTION_FOR_AUDIO
+    - MEDIA_STREAMER_PARAM_WEBRTC_RTP_TRANSCEIVER_DIRECTION_FOR_VIDEO
+    - MEDIA_STREAMER_PARAM_MAX_LATENESS
+    - MEDIA_STREAMER_PARAM_RTP_LATENCY
+    - media_streamer_node_decoded_ready_cb
+    - media_streamer_node_set_decoded_ready_cb
+    - media_streamer_node_unset_decoded_ready_cb
+    - media_streamer_webrtc_node_set_message_cb
+    - media_streamer_webrtc_node_unset_message_cb
+- Image Util
+  - Support for WebP has been added.
+  - Support for encoding for GIF and WebP animation has been added.
+  - Support for ARGB and GBRA for PNG encoding has been added.
+- Metadata Editor
+  - Support for WAV, FLAC, and OGG has been added.
+- Media Controller
+  - Result codes for command have been added.
+  - Three Playback states have been added. (Connection, Buffering, and Error)
+- Media Vision
+  - New enumerations for human pose landmark and body part have been added.
+  - API set to detect pose landmark, i.e., human body pose, has been added.
+- Recorder
+  - API set to decide video frame encoding has been added.
+  - Support for video scaling has been added.
+    - It is only for single video stream supported device.
+
+
+### Network and Connectivity
+
+#### New and changed features
+
+- Connection API
+  - Following API set to require new partner level privilege has been changed.
+    - New privilege: http://tizen.org/privilege/network.route
+    - Privilege level: partner
+    - Affected API set
+      - connection_add_route
+      - connection_remove_route
+      - connection_add_route_ipv6
+      - connection_remove_route_ipv6
+      - connection_add_route_entry
+      - connection_remove_route_entry
+- Bluetooth mesh network
+  - Bluetooth framework has been extended to support Bluetooth mesh network standard.
+    - Platform API for Bluetooth mesh network features has been added.
+    - Open source bluez module has been upgraded to support Bluetooth mesh network.
+    - Tizen Bluetooth framework has been extended to control bluez’s mesh daemon.
+  - Tizen devices will be able to do Bluetooth mesh network provisioning and control devices like smart blubs, which support Bluetooth mesh network standard.
+- Wi-Fi multiple interface support
+  - Network modules have been extended to support devices with multiple Wi-Fi network interfaces.
+    - connman and net-config have been extended to identify a specific Wi-Fi network interface for its operations like scan and connect.
+    - Platform API for specifying Wi-Fi network interface for each operation has been added for connection and wifi-manager API.
+  - Public connection and wifi-manager API are backward compatible by introducing a concept of default Wi-Fi network interface.
+- UWB (Ultra wideband) ranging
+  - Platform API for UWB has been added.
+  - UWB manager daemon has been added.
+  - It has been tested with Decawave’s dwm1001 module on RPI4.
```

Comment:
Please check if it is aligned properly, I preassume it should be right-aligned. Correct me if I am wrong. 
Also, check for technical correctness.
```suggestion
  - UWB manager daemon has been tested with Decawave’s dwm1001 module on RPI 4.

Reply:
will accept.